File: VB/CustomPrintPreview/Grid/CustomGridViewInfoRegistrator.vb
Imports DevExpress.XtraGrid
Imports DevExpress.XtraGrid.Views.Base
Imports DevExpress.XtraGrid.Registrator

Namespace CustomPrintPreview.Grid
    Public Class CustomGridViewInfoRegistrator
        Inherits GridInfoRegistrator

        Public Overrides ReadOnly Property ViewName() As String
            Get
                Return CustomGridView.CustomName
            End Get
        End Property

        Public Overrides Function CreateView(ByVal grid As GridControl) As BaseView
            Return New CustomGridView(grid)
        End Function
    End Class
End Namespace

File: CS/CustomPrintPreview/Grid/CustomGridViewInfoRegistrator.cs
using DevExpress.XtraGrid;
using DevExpress.XtraGrid.Views.Base;
using DevExpress.XtraGrid.Registrator;

namespace CustomPrintPreview.Grid {
    public class CustomGridViewInfoRegistrator :GridInfoRegistrator {
        public override string ViewName {
            get { return CustomGridView.CustomName; }
        }

        public override BaseView CreateView(GridControl grid) {
            return new CustomGridView(grid);
        }
    }
}
